<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>971</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabP">
     <attribute name="title">
      <string>Preprocessing</string>
     </attribute>
     <widget class="QPushButton" name="PpushButtonSelectFolder">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Folder</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="PgraphicsView">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>30</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="interactive">
       <bool>true</bool>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::MinimalViewportUpdate</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="PgraphicsViewEdge">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>250</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFS">
     <attribute name="title">
      <string>Features Selection</string>
     </attribute>
     <widget class="QPushButton" name="FSpushButtonOpenFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Open File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FSpushButtonSaveFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save File</string>
      </property>
     </widget>
     <widget class="QComboBox" name="FScomboBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>69</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Feature number:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="FSradioButtonFisher">
      <property name="geometry">
       <rect>
        <x>251</x>
        <y>50</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Fisher</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="FSradioButtonSFS">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>82</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>SFS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FSpushButtonCompute">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Compute</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="FStextBrowserDatabaseInfo">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabC">
     <attribute name="title">
      <string>Classifiers</string>
     </attribute>
     <widget class="QPushButton" name="CpushButtonOpenFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Open File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CpushButtonSaveFile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Save File</string>
      </property>
     </widget>
     <widget class="QComboBox" name="CcomboBoxK">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>61</y>
        <width>69</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ClabelK">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>20</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;k:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CpushButtonTrain">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Train</string>
      </property>
     </widget>
     <widget class="QLabel" name="ClabelClassifiers">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Classifiers:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="CcomboBoxClassifiers">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>69</width>
        <height>31</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="CpushButtonExecute">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
     <widget class="QLabel" name="ClabelTraningPart">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Training part:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="CtextBrowser">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>261</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxTrainingPart">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>104</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Basic">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>110</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Basic</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Bootstrap">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>140</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bootstraping</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_CrossValidation">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>170</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cross - validation</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelTrainUnit">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
     <widget class="QComboBox" name="CcomboBootstrapK">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>70</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ClabelBootstrapK">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; K:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
